test_genetic_algorithm_konami_code.py

```diff
@@ -1,35 +1,32 @@
 import random
```

Comment:
Ok we just need to break this test script to correspond to how you've broken up the original main script, and then also should probably add some tests for the parser, but after that we can probably merge, and then I might push a few more commits after that.

Reply:
I'm going to push to my fork an updated test file with some tests for the parser stuff. 

Now, for the existing tests you created, I already changed the imports to use the new files, and the tests are still passing.

Reply:
ya I just mean, I usually like to have the test scripts broken up so that they correspond to the files of the functions. When it was all just `genetic_algorithm_konami_code.py` it made sense to put all the tests in `test_genetic_algorithm_konami_code.py`, but now that it's broken up across main, player, parser, and helpers, the tests should reflect that in `test_player`, `test_helpers`, etc.

After that it's good to go :).